test/rule-matches/color-contrast-matches.js

```diff
@@ -238,6 +238,22 @@ describe('color-contrast-matches', function() {
 		assert.isFalse(rule.matches(target, axe.utils.getNodeFromTree(target)));
 	});
 
+	it("should not match an aria-disabled input's label - explicit label", function() {
+		fixture.innerHTML =
+			'<label for="t1">Test</label><input type="text" id="t1" aria-disabled="true">';
+		var target = fixture.querySelector('label');
+		axe.testUtils.flatTreeSetup(fixture);
+		assert.isFalse(rule.matches(target, axe.utils.getNodeFromTree(target)));
+	});
+
+	it("should not match a parent aria-disabled input's label - explicit label", function() {
+		fixture.innerHTML =
+			'<label for="t1">Test</label><div aria-disabled="true"><input type="text" id="t1"></div>';
```

Comment:
Can you explain why this should fail? I don't think aria-disabled cascades.

Reply:
I just it from the first few lines of the code that checks that the node or its parent has `aria-disabled` https://github.com/dequelabs/axe-core/blob/develop/lib/rules/color-contrast-matches.js#L6-L11.

Reply:
Opened #1766 to look into this